CONTRIBUTING.md

```diff
@@ -1,18 +1,22 @@
 # Contributing
 
-Thanks for your interest in contributing to `react-tailwindcss-datepicker`! Please take a moment to review this document **before submitting a pull request**.
+Thanks for your interest in contributing to `react-tailwindcss-datepicker`! Please take a moment to
```

Comment:
I was getting complaints from prettier until I reran the formatting. This was likely committed without running prettier?